### DISCUSSION TITLE
Resizing the vertical text area

### DISCUSSION THREAD

**Discussion by paridhikhaitan** [ueberdosis/tiptap#762] [General Questions] [answered]
I was wondering how a user could resize the tip tap text area manually by dragging it.

**Comment by hanspagel** [comment, marked as answer]
Try to add the following CSS to the `div` you want to be resizeable:

```css
resize: both;
overflow: hidden;
```

Hope that helps!

**Reply by paridhikhaitan** [reply]
Yes it did! Thank you so much for your help.
